openc3/lib/openc3/conversions/conversion.rb

```diff
@@ -30,12 +30,15 @@ class Conversion
     attr_reader :converted_bit_size
     # @return [Integer] The size in bits of the converted array value
     attr_reader :converted_array_size
+    # @return [] The Array of arguments passed in
+    attr_reader :args
```

Comment:
Probably rename to params.  If you look at the as_json method in a lot of our existing conversion classes they already are reporting their arguments as 'params'